sentry_sdk/api.py

```diff
@@ -37,87 +39,116 @@ def overload(x):
 
 # When changing this, update __all__ in __init__.py too
 __all__ = [
+    "add_breadcrumb",
     "capture_event",
-    "capture_message",
     "capture_exception",
-    "add_breadcrumb",
+    "capture_message",
     "configure_scope",
-    "push_scope",
+    "continue_trace",
     "flush",
-    "last_event_id",
-    "start_span",
-    "start_transaction",
-    "set_tag",
+    "get_baggage",
+    "get_client",
+    "get_current_span",
+    "get_traceparent",
+    "is_initialized",
+    "isolation_scope",
+    "new_scope",
+    "push_scope",
     "set_context",
     "set_extra",
-    "set_user",
     "set_level",
     "set_measurement",
-    "get_current_span",
-    "get_traceparent",
-    "get_baggage",
-    "continue_trace",
+    "set_tag",
+    "set_user",
+    "start_span",
+    "start_transaction",
 ]
 
 
-def hubmethod(f):
+def scopemethod(f):
     # type: (F) -> F
     f.__doc__ = "%s\n\n%s" % (
-        "Alias for :py:meth:`sentry_sdk.Hub.%s`" % f.__name__,
-        inspect.getdoc(getattr(Hub, f.__name__)),
+        "Alias for :py:meth:`sentry_sdk.Scope.%s`" % f.__name__,
+        inspect.getdoc(getattr(Scope, f.__name__)),
     )
     return f
 
 
-def scopemethod(f):
+def clientmethod(f):
     # type: (F) -> F
     f.__doc__ = "%s\n\n%s" % (
-        "Alias for :py:meth:`sentry_sdk.Scope.%s`" % f.__name__,
-        inspect.getdoc(getattr(Scope, f.__name__)),
+        "Alias for :py:meth:`sentry_sdk.Client.%s`" % f.__name__,
+        inspect.getdoc(getattr(Client, f.__name__)),
     )
     return f
 
 
-@hubmethod
+def is_initialized():
+    # type: () -> bool
+    """
+    .. versionadded:: 2.0.0
+
+    Returns whether Sentry has been initialized or not.
+
+    If a client is available and the client is active
+    (meaning it is configured to send data) then
+    Sentry is initialized.
+    """
+    return Scope.get_client().is_active()
+
+
+@scopemethod
+def get_client():
+    # type: () -> BaseClient
+    return Scope.get_client()
+
+
+@scopemethod
 def capture_event(
     event,  # type: Event
     hint=None,  # type: Optional[Hint]
     scope=None,  # type: Optional[Any]
     **scope_kwargs  # type: Any
 ):
     # type: (...) -> Optional[str]
-    return Hub.current.capture_event(event, hint, scope=scope, **scope_kwargs)
+    return Scope.get_current_scope().capture_event(
```

Comment:
Can we make the `capture_X` methods static?

It seems weird to have to use a specific scope (`get_current_scope` here) as a sort of proxy to call `capture_X` when the `capture_X` methods don't actually do anything with the scope they've been called on and ignore it completely -- they just perform a merge of the scopes in the current context and then capture stuff on the resulting merged scope.

Reply:
I do not think so. Francesco made a comment in one of our sync meetings, that for potel we need to also check "self" for data when merging the scope data.

Reply:
yeah but if you don't want to make it static, you should use the self scope in your implementation 
https://github.com/getsentry/sentry-python/blob/e556a1760dd21d386c4cc555f4cb517ce662434c/sentry_sdk/scope.py#L1045
and 
https://github.com/getsentry/sentry-python/blob/e556a1760dd21d386c4cc555f4cb517ce662434c/sentry_sdk/scope.py#L301

Reply:
imo using `Scope.get_client` which walks through all 3 variables (global/isolate/current) when you already on a scope object with a client is entirely unnecessary in all of these `capture_*/start_transaction` calls. You could just use `self.client`.

https://github.com/getsentry/sentry-python/blob/e556a1760dd21d386c4cc555f4cb517ce662434c/sentry_sdk/scope.py#L908

Reply:
this is also what JS is doing
https://github.com/getsentry/sentry-javascript/blob/6cfa68db484d5908960eb75b1e7feb3c1ae9c173/packages/core/src/scope.ts#L653

Reply:
Made it like it is implemented in JS. Global, Isolation, Current and then self are merged together.